packages/zostso/src/constants/ZosTso.constants.ts

```diff
@@ -20,7 +20,7 @@ export class TsoProfileConstants {
         aliases: ["a"],
         description: "Your z/OS TSO/E accounting information.",
         type: "string",
-        required: true,
+        required: false,
```

Comment:
I'd like to do some more testing around this before merging 😋 
Otherwise, we could revert this line change and just move my comment to a new issue 😋 
That way we don't have to wait any longer 😅